README.md

```diff
@@ -1,122 +1,209 @@
-# Instana Go Collector
+# IBM Instana Go Tracer
 
-![Instana, an IBM company](https://user-images.githubusercontent.com/203793/135623131-0babc5b4-7599-4511-8bf0-ce05922de8a3.png)
+<!-- ![Instana, an IBM company](https://user-images.githubusercontent.com/203793/135623131-0babc5b4-7599-4511-8bf0-ce05922de8a3.png) -->
+[<img src="https://user-images.githubusercontent.com/203793/135623131-0babc5b4-7599-4511-8bf0-ce05922de8a3.png" width="410"/>](135623131-0babc5b4-7599-4511-8bf0-ce05922de8a3.png)
```

Comment:
I wonder whether we need to use the new branding logo.

Reply:
hmmm good point! I will try to find out.

Reply:
afaics, we are the only ones adding an image here.
I am tempted to ditch this image completely. Any thoughts?

Reply:
Personally I like that some visual branding is present. It looks cool to me.. But I believe keeping this branding might lead us to problem if someone notices it. So I think for the time being we can remove it until we have a proper one.